internal/spectest/spectest.go

```diff
@@ -31,5 +33,26 @@ func FindJSONFilesInDir(t *testing.T, dir string) []string {
 		files = append(files, entry.Name())
 	}
 
+	if len(files) == 0 {
+		t.Fatalf("no JSON files found in %q", dir)
+	}
+
 	return files
 }
+
+// Path returns the absolute path to the given specifications repo file or
+// subdirectory.
```

Comment:
Optional: Consider describing how Path uses the PROJECT_DIRECTORY env var in the comment.

```suggestion
// Path returns the absolute path to the given specifications repo file or
// subdirectory.
//
// If the PROJECT_DIRECTORY environment variable is set, Path uses that to find
// the repo root path. Otherwise, it falls back to using the call stack to find
// the repo root path. Path panics if it can't find the repo root path.
```